### PR TITLE
DOC update - Mfa for the management console

### PR DESCRIPTION
**Pull request by divyaamunugama**

### CODE REVIEW

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Configuring multi-factor authentication for the management console
+
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:2]: need an intro here

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+## Prerequisites
+You need to [create a service provider](../applications/register-sp.md).
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:4]:
```suggestion
You need to [create a service provider]({{base_path}}/guides/applications/register-sp).
```

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Try it out
+
+1. Start the Identity Server.
+
+2. Access the following URL: `https://localhost:9443/`
+
+3. Enter your user credentials for the first authentication step.
+
+    ![basic-authentication](../../assets/img/guides/basic-authentication-mfa.png)
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:88]: let's use base path link

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    | Assertion Consumer URLs   | https://localhost:9443/acs    |
+    | Enable Response Signing   | Selected  |
+
+5. Click **Register** to save the configurations.
+
+### Local and outbound Authentication configurations
+
+To configure local and outbound authentication for the management console:
+
+1. Expand the **Local and Outbound Authentication Configuration** section and click **Advanced Configuration**.
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:41]:
```suggestion
1. Expand **Local and Outbound Authentication Configuration** and click **Advanced Configuration**.
```

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    1. [Inbound authentications configurations](#inbound-authentications-configurations)
+    2. [Local and outbound Authentication configurations](#local-and-outbound-authentication-configurations)
+    3. [Local configurations on `deployment.toml`](#local-configurations-on-deploymenttoml)
+
+
+### Inbound authentications configurations
+
+To configure inbound authentication for the management console:
+
+3. Expand the **Inbound Authentication Configuration > SAML2 Web SSO Configuration** section and click **Configure**.
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:25]: numbering

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+To configure MFA for the management console, initially:
+
+1. On the management console, go to **Main** > **Identity** > **Service Providers** > **List**.
+
+2. Click **Edit** on the service provider you wish to configure.
+
+    Configuration of MFA for the management console contains three configuration sections as follows:
+
+    1. [Inbound authentications configurations](#inbound-authentications-configurations)
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:16]: Should these be numbers or bullets? 
Should these be followed in sequence?

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+5. Enable the following options:
+    - Use tenant domain in local subject identifier
+    - Use user store domain in local subject identifier
+
+    !!! info
+        Leave the default configurations as it is.
+
+6. Click **Update** to save the configurations.
+
+### Local configurations on `deployment.toml`
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:62]: This should be a level 2 heading - `Configure WSO2 IS` or `Configure the server`

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
+# Configuring multi-factor authentication for the management console
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:1]: Can't we move this to the Management Console section under Setup?

Comment (divyaamunugama) [reply en/docs/guides/mfa/mfa-for-management-console.md:1]: +1 will move it

File: en/docs/guides/mfa/mfa-for-management-console.md (hunk 6 above)
Comment (divyaamunugama) [reply en/docs/guides/mfa/mfa-for-management-console.md:16]: 1 and 2 are okay to be interchanged, but 3 should be done as the last step

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+ This guide walks you through configuring multi-factor authentication for the Identity Server's Management Console.
+
+ MFA configuration for the management console contains two configuration sections as follows:
+
+1. [Configure the service provider](#configure-the-service-provider)
+    1. [Inbound authentications configurations](#inbound-authentications-configurations)
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:8]: In this case, the two sub points are not sequential yeah? Then, they can be bullets and not numbers.

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+## Configure WSO2 IS
+
+To configure the local configuration on the IS:
+
+1. Shut down the Identity server, and open the `deployment.toml` file in the `<IS_HOME>/repository/conf/` directory.
+
+2. Add the following configuration to the `deployment.toml` file:
+
+    ``` javascript
```
Comment (nilminiwso2) [inline en/docs/guides/mfa/mfa-for-management-console.md:73]:
```suggestion
    ``` toml
```

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
+# Configuring multi-factor authentication for the management console
+
+ This guide walks you through configuring multi-factor authentication for the Identity Server's Management Console.
+
+ MFA configuration for the management console contains two configuration sections as follows:
+
+1. [Configure the service provider](#configure-the-service-provider)
+    1. [Inbound authentications configurations](#inbound-authentications-configurations)
+    2. [Local and outbound Authentication configurations](#local-and-outbound-authentication-configurations)
```
Comment (divyaamunugama) [inline en/docs/guides/mfa/mfa-for-management-console.md:9]:
```suggestion
    - [Local and outbound Authentication configurations](#local-and-outbound-authentication-configurations)
```

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+2. [Configure the server](#local-configurations-on-deploymenttoml)
+
+## Prerequisites
+
+You need to [create a service provider]({{base_path}}/guides/applications/register-sp).
+
+## Configure the service provider
```
[the remark this answers is not in the document]
Comment (divyaamunugama) [reply en/docs/guides/mfa/mfa-for-management-console.md:16]: Yes, in this case, the management console is considered a normal application on the IS. 

The application specifications like the `Assertion Consumer URLs` is used to identify this application as the management console.

Other than that this has normal SP configurations.

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+ MFA configuration for the management console contains two configuration sections as follows:
+
+1. [Configure the service provider](#configure-the-service-provider)
+    - [Inbound authentications configurations](#inbound-authentications-configurations)
+    - [Local and outbound authentication configurations](#local-and-outbound-authentication-configurations)
+2. [Configure the server](#local-configurations-on-deploymenttoml)
+
+## Prerequisites
+
+You need to [create a service provider]({{base_path}}/guides/applications/register-sp).
```
Comment (divyaamunugama) [inline en/docs/guides/mfa/mfa-for-management-console.md:14]:
```suggestion
You need to [create a service provider]({{base_path}}/guides/applications/register-sp) to represent the Management Console on the Identity Server. You can use any name as the service provider name.
```

File: en/docs/guides/mfa/mfa-for-management-console.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+
+## Configure the service provider
+
+To configure MFA for the management console, initially:
```
Comment (divyaamunugama) [inline en/docs/guides/mfa/mfa-for-management-console.md:18]:
```suggestion
To configure MFA for the service provider which represents the management console, initially:
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**